### PR TITLE
JBDS-3868: Add new confirmation page

### PR DESCRIPTION
**Pull request by mohitsuman**
[JBDS-3868](https://issues.jboss.org/browse/JBDS-3868) Document (and check) required disk space before beginning install

image

### CODE REVIEW

**Comment by mohitsuman** [comment]
test this

**Comment by dgolovin** [comment]
I have fixed unit tests, but UI tests still requires attention.

File: test/ui/confirm-test.js [hunk 1]
```diff
@@ -17,128 +15,68 @@ for (let key in requirements) {
...
-};
-
-describe('Confirm page', function confimPage() {
-  let confirmForm, detectionInfo, nextButton;
+describe('Confirmation page', function() {
+  let installsize, downloadSize, backButton, nextButton, cancelButton, browseButton, confirmNote, summaryHeading;
 
   beforeAll(function() {
-    browser.setLocation('confirm')
+    browser.setLocation('selection')
```
Comment (jrichter1) [inline test/ui/confirm-test.js]: If you want to run this test after selection has been done, you can just add it to https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/protractor-conf.js#L30 after the selection test. It runs the files in sequence.

File: test/ui/confirm-test.js [hunk 2]
```diff
@@ -17,128 +15,68 @@ for (let key in requirements) {
...
       .then(function() {
-        confirmForm = element(By.id('confirmForm'));
-        detectionInfo = element(By.id('detection-info'));
-        nextButton = element(By.id('confirm-install-btn'));
+        return browser.setLocation('confirmation');
```
Comment (jrichter1) [inline test/ui/confirm-test.js]: The URI is just 'confirm' - this makes the tests fail actually :)

Comment (mohitsuman) [reply test/ui/confirm-test.js]: Updated in the latest commit.

File: test/ui/confirm-test.js (hunk 1 above)
Comment (mohitsuman) [reply test/ui/confirm-test.js]: yes will add the changes in the protractor-conf.js.

**Comment by mohitsuman** [comment]
test this

**Comment by mohitsuman** [comment]
test this

**Pull request merged**